File: app/src/main/java/com/ivantha/mobileatm/fragment/SettingsFragment.java
package com.ivantha.mobileatm.fragment;

import android.os.Bundle;
import android.support.v7.preference.PreferenceFragmentCompat;

import com.ivantha.mobileatm.R;

public class SettingsFragment extends PreferenceFragmentCompat {

    public static SettingsFragment newInstance() {
        SettingsFragment fragment = new SettingsFragment();
        return fragment;
    }

    @Override
    public void onCreatePreferences(Bundle bundle, String s) {
        // Load the Preferences from the XML file
        addPreferencesFromResource(R.xml.settings);
    }

}

File: app/src/main/java/com/ivantha/mobileatm/fragment/SettingsFragment.kt
package com.ivantha.mobileatm.fragment

import android.os.Bundle
import android.support.v7.preference.PreferenceFragmentCompat

import com.ivantha.mobileatm.R

class SettingsFragment : PreferenceFragmentCompat() {

    override fun onCreatePreferences(bundle: Bundle?, s: String?) {
        // Load the Preferences from the XML file
        addPreferencesFromResource(R.xml.settings)
    }

    companion object {

        fun newInstance(): SettingsFragment {
            return SettingsFragment()
        }
    }

}
